controllers/errors.controller.js

```diff
@@ -23,3 +23,25 @@ exports.handle500 = (error, req, res, next) => {
   console.log(error);
   res.status(500).send("whoops :(");
 };
+
+exports.handleCustom = (error, req, res, next) => {
+  if (error.status && error.msg) {
+    
+    res.status(error.status).send({ msg: error.msg });
+  } else next(error);
+};
+
+exports.handle404 = (req, res) => {
+  res.status(404).send({ msg: "not found" });
+};
+
+exports.handle400 = (error, req, res, next) => {
+  if (
+    error.code === "23502" ||
```

Comment:
you still have both the 23502 and 23503 err codes in here. Only one of them is a 400 the other is a 404